Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -649,11 +649,16 @@ private OrderResponse PreOrderChecksImpl(SubmitOrderRequest request)
             }
 
             //Ordering 0 is useless.
-            if (request.Quantity == 0 || request.Symbol == null || request.Symbol == QuantConnect.Symbol.Empty || Math.Abs(request.Quantity) < security.SymbolProperties.LotSize)
+            if (request.Quantity == 0)
```

Comment:
Removed two checks here; do you think they'll not be needed? 
"request.Symbol == null || request.Symbol == QuantConnect.Symbol.Empty"

Reply:
Before that if-statement, there is this one: 
`if (!Securities.TryGetValue(request.Symbol, out security))`
If `request.Symbol == null`, it will throw an exception.
If `request.Symbol == QuantConnect.Symbol.Empty`, it will be true, since Securities keys should not be `Symbol.Empty`.